taca/backup/backup.py

```diff
@@ -279,11 +314,19 @@ def pdc_put(cls, run):
                 continue
             flag = open(run.flag, 'w').close()
             with filesystem.chdir(run.path):
+                if bk.file_in_pdc(run.zip_encrypted, silent=False) or bk.file_in_pdc(run.dst_key_encrypted, silent=False):
+                    logger.warn("Seems like files realted to run {} already exist in PDC, check and cleanup".format(run.name))
+                    bk._clean_tmp_files([run.flag])
+                    continue
+                logger.info("Sending file {} to PDC".format(run.zip_encrypted))
                 if bk._call_commands(cmd1="dsmc archive {}".format(run.zip_encrypted), tmp_files=[run.flag]):
+                    time.sleep(15) # give some time just in case 'dsmc' needs to settle
```

Comment:
there is no way to check if dsmc is ready? So that instead of waiting 15 sec you can check if it is ready and fail if after 60 sec it is not yet ready?